src/math.d.ts

```diff
@@ -0,0 +1,17 @@
+export interface MathConfig {
+	engine?: string;
+	lazyLoad?: () => Promise<any>;
+	outputType?: string;
+	forceOutputType?: boolean;
+	enablePreview?: boolean;
+	previewClassName?: string[];
+	popupClassName?: string[];
+	katexRenderOptions: { [key: string]: any };
+	keepOpen?: boolean;
+}
+
+declare module "@ckeditor/ckeditor5-core/src/editor/editorconfig" {
+	interface EditorConfig {
+		math?: MathConfig;
+	}
```

Comment:
what do you need this for? i dont see any usages in the webapp or inside the ckeditor5

Reply:
It is not used in ckeditor5, but it's needed in webapp because typescript would complain about missing types. Now ckeditor5 is imported without `@ts-ignore`
```
import ClassicEditor from 'ckeditor5'
```